Dockerfile-dev

```diff
@@ -8,6 +8,7 @@ ENV CW_API /api/v1
 ENV ESP_API https://data.emissionspathways.org/api/v1
 ENV S3_BUCKET_NAME climate-watch-dev
 ENV GOOGLE_ANALYTICS_ID UA-1981881-51
+ENV USER_REPORT_KEY 5fe46fa7-c7e6-40e6-8e48-37479c4599f1
```

Comment:
didn't they send a separate key for staging and one for prod? We should probably distinguish this on the Jenkinsfile.